packages/experimental/Avatar/FRNAvatar.podspec

```diff
@@ -16,11 +16,11 @@ Pod::Spec.new do |s|
 
   s.ios.deployment_target = "14.0"
   s.ios.source_files      = "ios/*.{swift,h,m}"
-  s.ios.dependency 'MicrosoftFluentUI/Avatar_ios', '0.6.1'
```

Comment:
Is there a reason the '/Avatar_ios/ part got removed

Reply:
@Saadnajmi had mentioned that it was added because we can use pod subspaces to manage specific components (please correct me if I'm wrong), but updating the pod version alone should be sufficient so we decided to remove it (also one less line to update in the future)

Reply:
Reason was it made it slightly simpler to help Sophia update FUA in FURN. We don't _need_ the subspecs, but it Is technically more correct to have them. In the end, I don't think it actually makes any difference to the binary size in the Nuget for these libraries anyway.